tests/test_tracing/test_base.py

```
@@ -0,0 +1,357 @@
+"""
+Tests for the base tracing interface.
+"""
+
+import pytest
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)